hyperspy/signal.py

```diff
@@ -4613,14 +4613,23 @@ def derivative(self, axis, order=1, out=None, **kwargs):
         """
         # rechunk was a valid keyword up to HyperSpy 1.6
         if "rechunk" in kwargs:
+            # We miss the deprecation cycle for 2.0
+            warnings.warn(
+                "The `rechunk` parameter is not used since HyperSpy 1.7 and"
```

Comment:
```suggestion
                "The ``rechunk`` parameter is not used since HyperSpy 1.7 and"
```

Reply:
This isn't rst formatting! 😉 Should we keep it as a single quotation mark?